src/data/wallets/wallet-data.ts

```diff
@@ -1373,23 +1373,23 @@ export const walletsData: WalletData[] = [
     new_to_crypto: true,
   },
   {
-    last_updated: "2022-08-31",
+    last_updated: "2024-09-26",
     name: "Enkrypt",
     image: EnkryptImage,
     brand_color: "#ffffff",
     url: "https://www.enkrypt.com",
     active_development_team: true,
     languages_supported: ["en"],
-    twitter: "https://twitter.com/enkrypt",
-    discord: "https://discord.com/invite/Gz8vBneyKe",
-    reddit: "https://www.reddit.com/r/enkryptcom/",
+    twitter: "https://x.com/enkrypt",
+    discord: "https://discord.com/invite/W8TrqJXS",
+    reddit: "https://www.reddit.com/r/MyEtherWallet/",
```

Comment:
- Reverting Discord invite to old (working) one
- You've accidentally change reddit to MEW link.

```suggestion
    discord: "https://discord.com/invite/Gz8vBneyKe",
    reddit: "https://www.reddit.com/r/enkryptcom/",
```